gradle-version/src/gradle-version.coffee

```diff
@@ -0,0 +1,20 @@
+# Description
+#   A hubot script that gets the current release version of Gradle
+#
+# Commands:
+#   hubot gradle version - gets the current release version of Gradle
+#
+# Author:
+#   Jonatan Ivanov <jonatan.ivanov@gmail.com>
+
+module.exports = (robot) ->
+  robot.respond /.*[Gg]radle version.*/, (response) ->
+    robot.http('https://services.gradle.org/versions/current').get() (err, res, body) ->
+      if err
+        response.reply "Encountered an error:\n#{err}"
+      else
+        statusCode = parseInt(res.statusCode)
+        if 200 <= statusCode and statusCode < 300
```

Comment:
Readability is a little off, with the fixed value being on the left, and then on the right

Reply:
I was thinking about that too, this way it seems like a between clause :)

Reply:
Fixed.